lib/xslt/tei_to_html/lib/formatting.xsl

```diff
@@ -1,51 +1,63 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xsl:stylesheet 
-  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
-  xmlns:tei="http://www.tei-c.org/ns/1.0"
-  xmlns:xs="http://www.w3.org/2001/XMLSchema"
-  xpath-default-namespace="http://www.tei-c.org/ns/1.0"
-  version="2.0"
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:tei="http://www.tei-c.org/ns/1.0" xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xpath-default-namespace="http://www.tei-c.org/ns/1.0" version="2.0"
   exclude-result-prefixes="xsl tei xs">
-  
+
+  <!-- ================================================ -->
+  <!--             GENERIC CLASS TEMPLATE               -->
+  <!-- ================================================ -->
+
   <!-- this template is used to generate a list of classes that 
     represent all the attributes and values on a tag for instance, 
     <fw type="pageNum test" place="top"> would render as "tei_fw_type_pageNum tei_fw_type_test tei_fw_place_top tei_fw" -->
   <!-- line returns or spaces are passed through to attributes so are removed -->
+
   <xsl:template name="add_attributes">
     <xsl:if test="@*">
       <xsl:for-each select="@*">
-        <xsl:variable name="att_prefix"><xsl:text>tei_</xsl:text><xsl:value-of select="../name()"/><xsl:text>_</xsl:text><xsl:value-of select="name()"/><xsl:text>_</xsl:text></xsl:variable>
-        <xsl:for-each select="tokenize(.,' ')"><xsl:value-of select="$att_prefix"/><xsl:value-of select="."/><xsl:text> </xsl:text></xsl:for-each>
+        <xsl:variable name="att_prefix">
+          <xsl:text>tei_</xsl:text>
+          <xsl:value-of select="../name()"/>
+          <xsl:text>_</xsl:text>
+          <xsl:value-of select="name()"/>
+          <xsl:text>_</xsl:text>
+        </xsl:variable>
+        <xsl:for-each select="tokenize(., ' ')">
+          <xsl:value-of select="$att_prefix"/>
+          <xsl:value-of select="."/>
+          <xsl:text> </xsl:text>
+        </xsl:for-each>
       </xsl:for-each>
-    </xsl:if><xsl:text>tei_</xsl:text><xsl:value-of select="name()"/>
+    </xsl:if>
+    <xsl:text>tei_</xsl:text>
+    <xsl:value-of select="name()"/>
   </xsl:template>
 
-<!-- ================================================ -->
-<!--                   ABBREVIATION                   -->
-<!-- ================================================ -->
+  <!-- ================================================ -->
+  <!--                   ABBREVIATION                   -->
+  <!-- ================================================ -->
+
+  <xsl:template match="choice[child::abbr]">
+    <a>
+      <xsl:attribute name="rel">
+        <xsl:text>tooltip</xsl:text>
+      </xsl:attribute>
+      <xsl:attribute name="class">
+        <xsl:text>abbr</xsl:text>
+      </xsl:attribute>
+      <xsl:attribute name="title">
+        <xsl:apply-templates select="expan"/>​ </xsl:attribute>
+      <xsl:apply-templates select="abbr"/>
+    </a>​ </xsl:template>
```

Comment:
do you know why this change and the change on the line with `expan` were altered? Seems like they lost their indentation here

Reply:
This is the default indentation behavior for oxygen. I'd like to just leave it because it will just go back if I ever use auto indent in the future